File: DNN Platform/Admin Modules/Dnn.Modules.ModuleCreator/Templates/VB/DAL2/templateInfo.vb
#Region "Copyright"

' 
' Copyright (c) _YEAR_
' by _OWNER_
' 

#End Region

#Region "Using Statements"

Imports System
Imports System.Web.Caching
Imports DotNetNuke.ComponentModel.DataAnnotations

#End Region

Namespace _OWNER_._MODULE_

    <TableName("_OWNER___MODULE_s")> _
    <PrimaryKey("_MODULE_ID")> _
    <Scope("ModuleID")> _
    <Cacheable("_MODULE_s", CacheItemPriority.[Default], 20)> _
    Public Class _MODULE_Info
        Public Property _MODULE_ID() As Integer
            Get
                Return m__MODULE_ID
            End Get
            Set
                m__MODULE_ID = Value
            End Set
        End Property
        Private m__MODULE_ID As Integer
        Public Property ModuleID() As Integer
            Get
                Return m_ModuleID
            End Get
            Set
                m_ModuleID = Value
            End Set
        End Property
        Private m_ModuleID As Integer
        Public Property Title() As String
            Get
                Return m_Title
            End Get
            Set
                m_Title = Value
            End Set
        End Property
        Private m_Title As String
        Public Property Description() As String
            Get
                Return m_Description
            End Get
            Set
                m_Description = Value
            End Set
        End Property
        Private m_Description As String
        Public Property IsActive() As Boolean
            Get
                Return m_IsActive
            End Get
            Set
                m_IsActive = Value
            End Set
        End Property
        Private m_IsActive As Boolean
        Public Property CreatedOnDate() As DateTime
            Get
                Return m_CreatedOnDate
            End Get
            Set
                m_CreatedOnDate = Value
            End Set
        End Property
        Private m_CreatedOnDate As DateTime
        Public Property CreatedByUserID() As Integer
            Get
                Return m_CreatedByUserID
            End Get
            Set
                m_CreatedByUserID = Value
            End Set
        End Property
        Private m_CreatedByUserID As Integer
    End Class
End Namespace

File: DNN Platform/Admin Modules/Dnn.Modules.ModuleCreator/Templates/C#/DAL2/templateInfo.cs
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
// See the LICENSE file in the project root for more information
#region Using Statements

using System;
using System.Web.Caching;
using DotNetNuke.ComponentModel.DataAnnotations;

#endregion

namespace _OWNER_._MODULE_
{

    [TableName("_OWNER___CONTROL_s")]
    [PrimaryKey("_CONTROL_ID")]
    [Scope("ModuleID")]
    [Cacheable("_CONTROL_s", CacheItemPriority.Default, 20)]
    public class _CONTROL_Info
    {
        public int _CONTROL_ID { get; set; }
        public int ModuleID { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public bool IsActive { get; set; }
        public DateTime CreatedOnDate { get; set; }
        public int CreatedByUserID { get; set; }
    }
}
